.scripts/overmind/status.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Quick status check for overmind processes
+# Usage: ./status.sh
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)